extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -327,7 +327,6 @@ private function uploadImage($upload) {
 	}
 
 	public function executeSaveSettings() {
-		wfProfileIn( __METHOD__ );
 		global $wgRequest;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)